cmd/errors.go

```diff
@@ -9,4 +9,6 @@ const (
 	ErrorReadOnlyRootFilesystemFalse
 	ErrorRunAsNonRootNIL
 	ErrorRunAsNonRootFalse
+	ErrorCapabilitiesNIL
+	ErrorCapabilitiesAddedOrNotDropped
```

Comment:
Not sure about this that we want to merge this two error condition or not? I remember having a discussion with @klautcomputing

Reply:
@csgregorian and I discussed that it's easier for now to just have one, get all the errors named in kubeaudit and then once this is done we will search for a way to support multiple errors in one result. That change could go hand in hand with the log level change and we could come up with a something that supports both multiple errors and different log levels.
But let's see about that.
I am fine to merge this as is and address it later.

Reply:
cool